urdf/include/urdf/model.h

```diff
@@ -51,10 +51,10 @@ namespace urdf{
 class Model: public ModelInterface
 {
 public:
-  /// \brief Load Model from TiXMLElement
-  bool initXml(TiXmlElement *xml);
-  /// \brief Load Model from TiXMLDocument
-  bool initXml(TiXmlDocument *xml);
+  /// \brief Load Model from XMLElement
+  bool initXml(tinyxml2::XMLElement *xml);
+  /// \brief Load Model from XMLDocument
+  bool initXml(tinyxml2::XMLDocument *xml);
```

Comment:
If we can retarget this pull request to ROS M, I'd be in favor of tinyxml2 not being visible in the APIs at all.